package.json

```diff
@@ -29,7 +29,8 @@
     "prettier-config": "github:boyum/prettier-config",
     "typescript": "^4.9.4",
     "unplugin-json-dts": "^1.2.0",
-    "vite": "3.0.9"
+    "vite": "3.0.9",
+    "sass": "^1.57.1"
```

Comment:
Can we move it between `prettier-config` and `typescript`? :)))) I prefer the alphabetical order

Reply:
Yes :))